torch/_lazy/__init__.py

```diff
@@ -0,0 +1,25 @@
+import torch._C._lazy
+
+
+def mark_step(device: torch.device = "lazy:0"):
+    # if xu.getenv_as('LTC_EMIT_STEPLOG', bool, False):
```

Comment:
@JackCaoG here's a great example to discuss.  How do you envision the mark_step API working when torch-xla is part of torch::lazy?  Do you envision your users would ultimately `import torch._lazy.mark_step`, once you get past the stages of migration?

note: I did not intend to stop supporting `_run_step_closures()` but I might consider landing without it until we actually add some test coverage / usage scenarios of it.

Generally, my approach would be to only land the minimum bindings/apis we know we need and then keep adding them, rather than just land a bunch up front..

Reply:
I remember the decision is to add the `GetDefaultDevice` API? It's just that we never find time to do that. Maybe we should add a comment above.

Reply:
I think we actually do have a 'GetDefaultDevice' API for the backend, but we assume it is static after initialization time.  Maybe what I didn't want to do was have 'set/get current device' and have some state changing.

I was also thinking about how to improve our configuration flags and backend init APIs.  If we provide a good way to configure the backend configuration (python binding and then ENV override for each flag) could you use one of those flags to set default device type?  Or do you need something more than that?

Reply:
This is the dummy stub we have in the staging branch: https://github.com/pytorch/pytorch/blob/8b8c69101c4da12df6dd6782906100a4f0e46779/lazy_tensor_core/lazy_tensor_core/csrc/init_python_bindings.cpp#L587.